### ISSUE TITLE
Publish as AppX package and add to Microsoft Store

### ISSUE THREAD

**Issue by vantezzen** [getferdi/ferdi#485]
Many Windows apps get distributed over the Microsoft Store - Window's App Store. This store allows users to more easily and more securely installs applications. Additionally, some versions of Windows - like Windows 10 S - don't even give the user the ability to run normal .exe apps anymore.

ElectronJS has published a guide on how we can get Ferdi onto the Microsoft Store: https://www.electronjs.org/docs/tutorial/windows-store-guide

This would require us to use the [electron-windows-store](https://github.com/felixrieseberg/electron-windows-store) to package our Windows build to the AppX format. We would also need to create a Microsoft Developer Account which costs a one-time fee of $19.

**Comment by vantezzen** [comment]
Adding this via https://github.com/getferdi/ferdi/tree/feat/appx - we would require #415 to test this setup though.

**Comment by kytwb** [comment]
@vantezzen Do we need electron-windows-store in the end? I think your branch re-use electron-builder right? Also, we should be careful to still keep auto-update available for NSIS. Let me know if I should go ahead and get the developer account or if you planned on doing it 🙏

**Comment by vantezzen** [comment]
Yes, I noticed that electeon-builder has AppX supper build-in so we don't need the extra package.
I used electrons process.windowsstore to disable updates, which should only be true if we are coming from the store.

I'll test AppX on our Windows VM and only after that I'll need the dev account - I'll notify you so you can set that up.
